### DISCUSSION TITLE
Alternative to `settings sync` extension for vs code

### DISCUSSION THREAD

**Discussion by vigneshbalu96** [Cyfrin/foundry-full-course-cu#1891] [General]
I came across the vs code course and thought of adding the extension of 
>  _**settings sync**_ 

but it was outdated and if anybody knows an alternate solution that would be helpful. Thanks!

**Comment by EngrPips** [comment]
Hmmm, if I find anything useful, I will share it. But what does `settings sync` do?

**Comment by vigneshbalu96** [comment]
Quick Solution here @EngrPips , vs code handles all the work for syncing all required fields instead of 3rd party like ```settings sync```.

You can check that in **Accounts** in vs code and turn it on. :smiley: 

The benefit of having the sync is it will take the existing configurations, snippets, shortcuts, settings and extensions put them into a file, so that we with that file we can able to use it with any instances of vs code.

**Reply by EngrPips** [reply]
Meaning if I don't have it turned on, I will always need to make configuration for every single instance of vs code that I initiate.
